evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -71,7 +71,7 @@ def test_all_estimators(has_minimal_dependencies):
     if has_minimal_dependencies:
         assert len((_all_estimators_used_in_search())) == 10
     else:
-        assert len(_all_estimators_used_in_search()) == 16
+        assert len(_all_estimators_used_in_search()) == 17
```

Comment:
My understanding is that this PR should only be adding the estimator, *not* to search which is handled by #1946? If that's true, this shouldn't be updated, correct?